internal/report/report.go

```diff
@@ -64,6 +64,14 @@ type Options struct {
 	Ratio               float64
 	Title               string
 	ProfileLabels       []string
+	Focus               string
```

Comment:
It might be simpler to have instead something like `ActiveFilters []string`, put there `key=value` entries like `focus=foo` and print them out as is. The code would be shorter and it would also be clearer that these are passed in just for display purposes, not to be used for the actual report construction.